src/content/examples/hi/14_Loading_And_Saving_Data/02_Table/description.mdx

```diff
@@ -1,15 +1,12 @@
 ---
 featuredImage: "../../../images/featured/14_Loading_And_Saving_Data-02_Table-thumbnail.png"
-title: Table
-one-line-description: Store data as comma-separated values.
+title: मेज़
```

Comment:
Use suuchi instead, this means physical table

Reply:
Found a better alternative तालिका